staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache_interval.go

```diff
@@ -162,15 +161,11 @@ func newCacheIntervalFromStore(resourceVersion uint64, store storeIndexer, getAt
 		if !ok {
 			return nil, fmt.Errorf("not a storeElement: %v", elem)
 		}
-		objLabels, objFields, err := getAttrsFunc(elem.Object)
```

Comment:
Given getAttrsFunc is no longer used in this function, let's also change the signature of this function.

Reply:
done